lib/screens/explore_screens/regions_screens/regions_main/components/region_main_app_bar.dart

```diff
@@ -12,7 +13,14 @@ class RegionMainAppBar extends StatelessWidget {
     return BlocBuilder<RegionBloc, RegionState>(
       builder: (context, state) {
         return SliverAppBar(
-          expandedHeight: 200.0,
+          actions: [
+            if (!state.isBrowseView)
```

Comment:
I dont think we need the "!" here

Reply:
Menu dots should be only present in full view not in brose view.